asv_bench/benchmarks/frame_ctor.py

```diff
@@ -118,4 +119,39 @@ def time_frame_from_range(self):
         self.df = DataFrame(self.data)
 
 
+class FromArrays:
+
+    goal_time = 0.2
+
+    def setup(self):
+        N_rows = 1000
+        N_cols = 1000
+        self.float_arrays = [np.random.randn(N_rows) for _ in range(N_cols)]
```

Comment:
Maybe use a random state here and below?

Reply:
In principle the values inside the array should have no influence, but also can't harm I suppose, will add

Reply:
Actually, random state already seems to be managed by

https://github.com/pandas-dev/pandas/blob/d308712c8edef078524b8a65df7cb74e9019218e/asv_bench/benchmarks/pandas_vb_common.py#L56-L60